### PR TITLE
🔨 feat : added dev layer to adhan package

### PR DESCRIPTION
**Pull request by Tauqeer-Ahmed-99**
Added : New Adhan NPM Package to calculate prayer times on the client side
Added : Extra dev layer to use adhan package as per our need

### CODE REVIEW

File: libs/prayer-info/src/parayer-times/enums.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export enum Madhab {
+  Hanafi = 'hanafi',
+  Shafi = 'shafi',
+}
+
+export enum Salaah {
+  Fajr = 'fajr',
+  Dhuhr = 'dhuhr',
+  Asr = 'asr',
+  Maghrrib = 'maghrib',
```
Comment (faran4engg) [inline libs/prayer-info/src/parayer-times/enums.ts:26]: Maghrib - removed extra R

Comment (Tauqeer-Ahmed-99) [reply libs/prayer-info/src/parayer-times/enums.ts]: removed : typ @faran4engg

**Comment by faran4engg** [comment]
Can we have test cases as well? I am not sure how to test these utils. During development, how did you test these, can you share them with everyone please?

**Comment by Tauqeer-Ahmed-99** [comment]
> Can we have test cases as well? I am not sure how to test these utils. During development, how did you test these, can you share them with everyone please?

Yes,
currently, I am working on a better file structure, and separating the helper functions.
Once I am done with that will start writing tests.

File: libs/prayer-info/package.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "name": "@islaamify-pro/prayer-info",
   "version": "0.0.1",
-  "type": "commonjs"
+  "type": "commonjs",
+  "dependencies": {
+    "adhan": "^4.4.3"
+  }
```
Comment (faran4engg) [inline libs/prayer-info/package.json:7]: it should be in the root

**Review by faran4engg — APPROVED** [review]

File: libs/prayer-info/package.json (hunk 2 above)
Comment (anask601) [reply libs/prayer-info/package.json]: done

**Review by anask601 — APPROVED** [review]

**Pull request merged**